submissions/iva-stasia/js-dom/js/script.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+const characters = [
+  {
+    id: 1,
+    name: 'Cowboy Bebop',
+    quote: '"Whatever happens, happens." — Spike Spiegel',
+    description1: 'Cowboy Bebop (カウボーイビバップ Kaubōi Bibappu) is a Japanese animated television series, movie and other media created by Shinichiro Watanabe. Set in 2071 and centered on the adventures of a gang of bounty hunters in space, the story delves into the unresolved issues of the protagonists\' past, exploring concepts such as existentialism, boredom, loneliness, and the influence of the past.',
```

Comment:
Set array to the property description. It needs if one of hero has more than three paragraphs in description or only one. And use methods `reduce` or `map/join` to view it.

Reply:
Done.